src/CoreWCF.Templates/src/templates/CoreWCFService/.template.config/template.json

```diff
@@ -97,6 +97,10 @@
       "description": "The target framework for the project.",
       "datatype": "choice",
       "choices": [
+        {
+          "choice": "net8.0",
+          "description": "Target net7.0"
```

Comment:
Need to update the description to net8.0

Reply:
done